src/v2/api/responses/EventsResponse.ts

```diff
@@ -29,4 +29,23 @@ export class GeneralEventsResponse {
 
   @ApiProperty()
     startTime: Date;
+}
+
+export class TelegramGeneralEventsResponse {
+  @ApiProperty({
+    type: [GeneralEventInfoResponse],
+  })
+    events: GeneralEventInfoResponse[];
+}
+
+export class FortnightGeneralEventsResponse {
+  @ApiProperty({
+    type: [GeneralEventInfoResponse],
+  })
+    eventsOfFirstWeek: GeneralEventInfoResponse[];
+
+  @ApiProperty({
+    type: [GeneralEventInfoResponse],
+  })
+    eventsOfSecondWeek: GeneralEventInfoResponse[];
```

Comment:
In this case I`d rather shorten: 
* `eventsOfFirstWeek` to `firstWeekEvents`;
* `eventsOfSecondWeek` to `secondWeekEvents`.